other-components-docker.yml

```
@@ -4,3 +4,5 @@
 services:
     mtls-ambassador:
         image: registry.mender.io/mendersoftware/mtls-ambassador:mender-master
+    mender-ci-workflows:
+        image: registry.mender.io/mendersoftware/mender-ci-tools:master
```

This image will go into Docker Hub, so it should be:
```suggestion
        image: mendersoftware/mender-ci-tools:master
```

On top of it, we'll use the `mender-1.2.3` kind of tags, so:
```suggestion
        image: mendersoftware/mender-ci-tools:mender-master
```